### ISSUE TITLE
Move a11y strings to en json strings file

### ISSUE THREAD

**Issue by zepumph** [phetsims/molarity#205]
This is the next step in the a11y string conversion process. previous issues include https://github.com/phetsims/rosetta/issues/214 and https://github.com/phetsims/chipper/issues/795

This is the first simulation that we are doing with this, and so I will work closely with @jbphet to make sure that everything works well. Since molarity is already in RC, my hope is to do this conversion in one commit on master, and then be able to cherry-pick it easily over to the rc branch.

**Comment by zepumph** [comment]
This was done in two commits above. There were a couple of assertions that needed to be fixed before committing this. I also spent a bit of time reorganizing the structure to try to encapsulate the comments, and then I removed the comments. I will now cherry-pick these into the rc branch.

**Comment by zepumph** [comment]
After discussing with @jbphet, we are going to create another rc immediately so that @jbphet can test this with a local copy of rosetta before publication.

**Comment by zepumph** [comment]
Alright @jbphet, the a11y strings have been integrated into the `json` strings file in https://phet-dev.colorado.edu/html/molarity/1.5.0-rc.2/phet/.  Please test this and let me know if you run into any problems with how rosetta ignores the a11y strings.

**Comment by jbphet** [comment]
Test complete, everything looks good.  As far as Rosetta is concerned, you should be good to publish this sim @zepumph.

For the record, here is a summary of how this testing was done:

+ Tested using a version of Rosetta that was running on my local machine, SHA = 539f81e1b30d9d68f9e819de9bc986f769a90985, which is close to what is currently live on the public website but probably a couple of minor commits ahead.
+ The code in `simData.getLiveSimUrl` was modified to use the RC link above as the English version of the sim from which the string keys are extracted when Molarity is the selected sim (this was obviously not committed).
+ Pointed Chrome to the locally running translation utility (http://localhost:16372/translate/ in my case), selected Molarity as the sim and Abkhazian (ab) as the locale.  Verified that none of the a11y strings appeared on the UI and that the output log contained entries about a11y strings being found in the sim but explicitly skipped.
+ Changed a couple of strings, verified that the "Test" feature worked.
+ Submitted the changed strings using the "Submit" button (Rosetta is configured to use the "tests" branch of babel when running on my machine), verified that the commit looked correct (commit: https://github.com/phetsims/babel/commit/67467a35eb8192568cf83a7bcb1e2f2b80f8e6c1).

Back to you @zepumph.

**Comment by zepumph** [comment]
This has been complete, and once this sim is published, we will note in https://github.com/phetsims/rosetta/issues/214.
